hooks/useMap.tsx

```diff
@@ -27,6 +27,10 @@ const useMap = () => {
       setIsVideoOpen(!isVideoOpen)
       return
     }
+    if (area.id === 'live-show') {
```

Comment:
Since each condition block (if) ends with return, there's no need for additional conditional checks after each action. By using an `else if structured, the function can break out after finding the matching condition, reducing checks and improving readability.